internal/service/image/pull.go

```diff
@@ -11,15 +11,17 @@ import (
 	"strings"
 	"time"
 
-	"github.com/containerd/containerd/remotes/docker"
+	"github.com/containerd/containerd/v2/core/remotes/docker"
 	cerrdefs "github.com/containerd/errdefs"
-	"github.com/containerd/nerdctl/pkg/imgutil/dockerconfigresolver"
+	"github.com/containerd/nerdctl/v2/pkg/imgutil/dockerconfigresolver"
 	dockertypes "github.com/docker/cli/cli/config/types"
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 
 	"github.com/runfinch/finch-daemon/pkg/errdefs"
 )
 
+const IndexServer = "https://index.docker.io/v1/"
```

Comment:
Is this used?

Reply:
This appears to be leftover from a refactor. Removing.